src/encoded/regulome_search.py

```diff
@@ -374,7 +389,15 @@ def parse_region_query(request):
         'format': format,
         'from': from_,
         'total': total,
-        'variants': {k: list(v) for k, v in sorted(variants.items())[from_:to_]},
+        'variants': [
+            {
+                'chrom': chrom,
+                'start': start,
+                'end': end,
+                'rsids': sorted(variants[(chrom, start, end)])
```

Comment:
ahh... here.

Reply:
Yes, here. Honestly, this doesn't have to be sorted. I just don't like arbitrary order. Maybe we should go back?

Reply:
I guess it's OK.